src/Engine/GameInput/SelectUnitAction.java

```diff
@@ -80,9 +80,15 @@ else if( actionType.shouldConfirm )
     return next;
   }
 
+  @Override
+  public void consider(GameActionSet actionSet)
+  {
+    myStateData.damagePopups = actionSet.getSelected().getDamagePopups(myStateData.gameMap);
```

Comment:
Not a fan of this; it makes it look like I'm about to commit an action before I actually am. Plus there's no (apparent) rhyme or reason for which option gets shown (and I don't think there's a good way to provide that).

Reply:
The one that gets shown is the one that would be selected first if you picked that option.
IMO it's nice because it indicates immediately which unit I'll attack if I just mash through. Also reduces button presses for checking out e.g. BBomb explosion places.

Reply:
Sure, I get that it gives an advance preview of the first option. Still don't like it. I'm used to seeing the attack stats or whatever just before committing, so having them show up earlier is confusing, and frankly the early preview really doesn't add much.

Re: Mashing: You either a) press to move and bring up the attack option, pause a moment to review it, then mash twice to commit, or you b) press twice to move and bring up the attack options, evaluate, then press once more.
You still have the same work, with the same essential pause to evaluate the option anyway. I'd rather keep a consistent interface language of showing action consequences just before commitment instead of muddying the water this way.